### PR TITLE
login looks good now

### PR DESCRIPTION
**Pull request by RandallCaler**

### CODE REVIEW

File: src/Forgot.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    props.handleSubmit(person);
+    setPerson({ email: ""});
+    //navigate("/home");
+  }
+
+  function Copyright(props) {
+    return (
+      <Typography variant="body2" color="text.secondary" align="center" {...props}>
+        {'Copyright © '}
+        <Link color="inherit" href="https://mui.com/">
```
Comment (Aorta3698) [inline src/Forgot.js]: should replace this with our URL

File: src/Login.js [hunk 2]
```diff
@@ -15,35 +30,124 @@ function Login(props) {
...
+    //props.handleSubmit(person);
     setPerson({ email: "", password: "" });
+    navigate("/home");
   }
 
+  function Copyright(props) {
+    return (
+      <Typography variant="body2" color="text.secondary" align="center" {...props}>
+        {'Copyright © '}
+        <Link color="inherit" href="https://mui.com/">
```
Comment (Aorta3698) [inline src/Login.js]: should replace this with our URL

File: src/Signup.js [hunk 3]
```diff
@@ -42,45 +57,156 @@ export default function AccountForm(props) {
...
-        value={person.password}
-        onChange={handleChange}
-      />
-      <input type="button" value="Submit" onClick={submitForm} />
-    </form>
+  function Copyright(props) {
+    return (
+      <Typography variant="body2" color="text.secondary" align="center" {...props}>
+        {'Copyright © '}
+        <Link color="inherit" href="https://mui.com/">
```
Comment (Aorta3698) [inline src/Signup.js]: should replace this with our URL

**Review by Aorta3698 — CHANGES_REQUESTED** [review]

**Review by anthony-yeo — DISMISSED** [review]
Looks good Randall, seems like Ash has some requested changes, just get those done and then merge

**Review by Aorta3698 — APPROVED** [review]
Great work @RandallCaler . Approved!

**Review by anthony-yeo — APPROVED** [review]

**Pull request merged**